docs/source/snippets.rst

```diff
@@ -0,0 +1,14 @@
+XRPL Code Snippets
+===================
+
+Code snippets to demonstrate basic usage of the xrpl-py library.
+
+.. toctree::
+   :maxdepth: 1
+
+   get_transaction
```

Comment:
Should there be links to each snippet, or should all the snippets just be on this page?

Reply:
Hm yeah I think all snippets on one page might be bit better and still have tabs on left bar for it